extensions/amp-form/0.1/amp-form.js

```diff
@@ -216,6 +221,29 @@ export class AmpForm {
   }
 
   /**
+   * Returns form data as an object.
+   * @return {!Object}
+   * @private
+   */
+  getFormAsObject_() {
```

Comment:
There's no way to actually access data inside a `FormData` object currently on webkit. It seems only Mozilla and Chrome supports methods like `FormData.entries()`. So no way to serialize FormData into an object in these browsers. 

We can choose to polyfill FormData support later on.